plugins/shell-proxy/shell-proxy.plugin.zsh

```diff
@@ -1,16 +1,37 @@
 #!/usr/bin/bash
```

Comment:
plz use [shellcheck](https://www.shellcheck.net) lint the shell script.
plz use [shfmt](https://github.com/mvdan/sh) format the shell script.

Reply:
`shellcheck` already passes, `shfmt` only does this:
```diff
diff --git a/plugins/shell-proxy/shell-proxy.plugin.zsh b/plugins/shell-proxy/shell-proxy.plugin.zsh
index bf0f4b0c..9d45b526 100644
--- a/plugins/shell-proxy/shell-proxy.plugin.zsh
+++ b/plugins/shell-proxy/shell-proxy.plugin.zsh
@@ -21,8 +21,8 @@ proxy() {
 
   # capture the output of the proxy script and bail out if it fails
   local output
-  output="$(SHELLPROXY_URL="$SHELLPROXY_URL" SHELLPROXY_CONFIG="$SHELLPROXY_CONFIG" "$proxy" "$1")" \
-  || return $?
+  output="$(SHELLPROXY_URL="$SHELLPROXY_URL" SHELLPROXY_CONFIG="$SHELLPROXY_CONFIG" "$proxy" "$1")" ||
+    return $?
 
   # evaluate the output generated by the proxy script
   source <(echo "$output")
```

Reply:
plz use `shfmt` the script, thanks